cluster/examples/kubernetes/nfs/pvc.yaml

```diff
@@ -2,10 +2,11 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: rook-nfs-pv-claim
+  annotations:
```

Comment:
help me understand why the beta annotation is being used instead of a storageClassName field?

Reply:
we can use storageclassName field as well.